### PR TITLE
task2 - Install git, docker and run jenkins on container + fix and update files on task 1 ( Bar Attar )

### PR DESCRIPTION
**Pull request by barattar1991**

### CODE REVIEW

File: bar/check_if_nginx_install.sh [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
-    echo "$1 installed"
-    sudo  yum -y autoremove
+if [[ $? -eq 0 ]]; then
+  read -p "nginx is already installed. uninstall it? [y/n]" user_ans
+  if [[ $user_ans == "y" ]]; then
+    echo "Uninstalling nginx..."
+    systemctl stop nginx.service
+    systemctl disable nginx.service
+    userdel -r nginx
+    rm -rf /etc/nginx
```
Comment (noamgu) [inline bar/check_if_nginx_install.sh]: You can combine it into single `rm -rf` command:
~~~
rm -rf folder1/ folder2/ folder3/
~~~

File: bar/bootstrap.sh [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+sudo yum -y install git
+ssh-keygen -t rsa -b 4096
+eval "$(ssh-agent -s)"
+ssh-add ~/.ssh/id_rsa
+ssh -T git@github.com
+sudo yum install -q -y yum-utils
+sudo yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+sudo yum -y install docker docker-ce docker-ce-cli containerd.io
```
Comment (noamgu) [inline bar/bootstrap.sh]: No need to inline `sudo` commands.
Make sure that the user has root privileges and add nice comment at the start of the script that saying that "this scripts need's to run as root"

File: bar/bootstrap.sh [hunk 3]
```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/bash
```
Comment (noamgu) [inline bar/bootstrap.sh:2]: Highly suggest to add some spaces between logical groups of code.
this make your code more readable and easily to maintain in the feature.

File: bar/check_if_nginx_install.sh (hunk 1 above)
Comment (barattar1991) [reply bar/check_if_nginx_install.sh]: fixed

File: bar/bootstrap.sh (hunk 2 above)
Comment (barattar1991) [reply bar/bootstrap.sh]: fixed

**Review by lmilbaum — APPROVED** [review]

**Pull request merged**